src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -958,16 +962,13 @@ class LicenceService(
     return parts[0].toInt() to parts[1].toInt()
   }
 
-  private fun inactivateInProgressLicenceVersions(licences: List<EntityLicence>, reason: String? = null) {
+  @Transactional
+  fun inactivateInProgressLicenceVersions(licences: List<EntityLicence>, reason: String? = null) {
     val licenceIds = licences.map { it.id }
     val licencesToDeactivate =
       licenceRepository.findAllByVersionOfIdInAndStatusCodeIn(licenceIds, listOf(IN_PROGRESS, SUBMITTED))
     if (licencesToDeactivate.isNotEmpty()) {
-      inactivateLicences(
-        licencesToDeactivate,
-        reason,
-        false,
-      )
+      inactivateLicences(licencesToDeactivate, reason, false)
```

Comment:
This was moaning that it was not going via the transactional proxy - it was only being called by transactional methods which I assumed would be fine but to remove doubt I made this method public/transactional